src/Infrastructure.Dapper/NotificationCenter/Repositories/NotificationStatusRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿#nullable enable
+using System.Data;
+using Bit.Core.NotificationCenter.Entities;
+using Bit.Core.NotificationCenter.Repositories;
+using Bit.Core.Settings;
+using Bit.Infrastructure.Dapper.Repositories;
+using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace Bit.Infrastructure.Dapper.NotificationCenter.Repositories;
+
+public class NotificationStatusRepository : BaseRepository, INotificationStatusRepository
+{
+    public NotificationStatusRepository(GlobalSettings globalSettings)
+        : this(globalSettings.SqlServer.ConnectionString, globalSettings.SqlServer.ReadOnlyConnectionString)
+    {
+    }
+
+    public NotificationStatusRepository(string connectionString, string readOnlyConnectionString)
+        : base(connectionString, readOnlyConnectionString)
+    {
+    }
+
+    public async Task<NotificationStatus?> GetByNotificationIdAndUserIdAsync(Guid notificationId, Guid userId)
+    {
+        await using var connection = new SqlConnection(ConnectionString);
+
+        return await connection.QueryFirstOrDefaultAsync<NotificationStatus>(
+            "[dbo].[NotificationStatus_GetByNotificationIdAndUserId]",
+            new { NotificationId = notificationId, UserId = userId },
+            commandType: CommandType.StoredProcedure);
+    }
+
+    public async Task<NotificationStatus> CreateAsync(NotificationStatus notificationStatus)
```

Comment:
With `src/Infrastructure.Dapper/Repositories/Repository.cs`, Create and Update does not need to be implemented from scratch

Reply:
The `NotificationStatus` table has a composite PK, so we can't inherit from `Repository.cs` like 
`Repository<NotificationStatus, Guid>,`.

There might be a way to directly call it but I have concerns since that class assume the table has an `Id` PK.